File: MODUL 2/C/PRAK202-2310817220011-ReginaSilvaMaharatini.c
#include <stdio.h>

int main(){
    float nilai1, nilai2, hasil;

    printf("Input\n");
    printf("Masukkan Nilai Pertama : ");
    scanf("%f", &nilai1);
    printf("Masukkan Nilai Kedua   : ");
    scanf("%f", &nilai2);

    hasil = nilai1 + nilai2;

    printf("\nOutput\n");
    printf("Hasil dari penjumlahan nilai pertama \"%g\" dan nilai kedua \"%g\" adalah \"%.2f\"\n", nilai1, nilai2, hasil);

    return 0;
}

File: MODUL 2/Python/PRAK202-2310817220011-ReginaSilvaMaharatini.py
nilai1 = float(input("Input\nMasukkan Nilai Pertama : "))
nilai2 = float(input("Masukkan Nilai Kedua   : "))

hasil = nilai1 + nilai2

output1 = (f"{int(nilai1)}" if nilai1.is_integer() else f"{nilai1:.2f}")

print("\nOutput")
print(f'Hasil dari penjumlahan nilai pertama "{output1}" dan nilai kedua "{nilai2}" adalah "{hasil:.2f}"')